### PR TITLE
Fixes #172 

### LINKED ISSUE #172: Easier sorting of questionnaires
Currently, questionnaires are sorted by an index, which is an attribute of each questionnaire. This attribute should not need to be edited directly, but through a drag'n'drop GUI.

### PR DESCRIPTION
**Pull request by hrantzsch**
fix #172

### CODE REVIEW

File: evap/evaluation/migrations/0005_auto_20150115_1717.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ('evaluation', '0004_evaluation_start_email_template'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='questionnaire',
+            name='index',
+            field=models.IntegerField(default=0, verbose_name='ordering index', blank=True),
```
Comment (janno42) [inline evap/evaluation/migrations/0005_auto_20150115_1717.py]: please remove ```blank=True```

File: evap/evaluation/models.py [hunk 2]
```diff
@@ -87,7 +88,7 @@ class Questionnaire(models.Model):
     teaser_en = models.TextField(verbose_name=_(u"teaser (english)"), blank=True, null=True)
     teaser = Translate
 
-    index = models.IntegerField(verbose_name=_(u"ordering index"))
+    index = models.IntegerField(verbose_name=_(u"ordering index"), default=0, blank=True)
```
Comment (janno42) [inline evap/evaluation/models.py]: please remove ```blank=True```

File: evap/staff/templates/staff_course_review.html [hunk 3]
```diff
@@ -27,9 +27,9 @@ <h3 class="panel-title">
                     </h3>
                 </div>
                 <div class="panel-body">
-                    {% for hidden in form.hidden_fields %}
-                        {{ hidden }}
-                    {% endfor %}
+{#                    {% for hidden in form.hidden_fields %}#}
```
Comment (janno42) [inline evap/staff/templates/staff_course_review.html]: please remove comments :)

Comment (janno42) [inline evap/staff/templates/staff_course_review.html]: and please remove the hidden_fields rendering also in ```staff_faq_index.html``` and ```staff_faq_section.html```

File: evap/staff/templates/staff_questionnaire_index.html [hunk 4]
```diff
@@ -17,3 +17,24 @@
...
 {% endblock %}
+
+{% block additional_javascript %}
+    <script type="text/javascript">
+    $('.questionnaire-table').sortable({
+                items: '.questionnaire',
+                handle: 'td:first',
+                tolerance: 'pointer',
+                update: function( event, ui ) {
+                    // update indices questionnaires in the table
```
Comment (janno42) [inline evap/staff/templates/staff_questionnaire_index.html]: please reword

File: evap/static/js/csrf_utils.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+// setup ajax sending csrf token
+$.ajaxSetup({
+	beforeSend: function(xhr, settings) {
+		if (!csrfSafeMethod(settings.type) && !this.crossDomain) {
+			xhr.setRequestHeader("X-CSRFToken", csrftoken);
+		}
+	}
+});
```
Comment (karyon) [inline evap/static/js/csrf_utils.js]: please add a comment from where you copied this stuff.

**Comment by janno42** [comment]
:+1:

**Comment by Bartzi** [comment]
closing this in favour of the newly created pull request #501

**Pull request closed without merging**